src/utils/transaction.ts

```diff
@@ -204,6 +204,51 @@ const fetchSwapAssetTransaction = async (
   }
 };
 
+const bridgeAssetToPlr = async (chainId: number, route: Route | null, sdk?: EtherspotSdk | null) => {
```

Comment:
add return type and maybe rename as `buildBridgeAssetToPlrTransactions` as this is not actually bridging, but returning some result which is transactions